src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.Season.StartingMonthHelp": "Le mois où cette saison commence chaque année.",
+  "FSC.Configuration.Season.StartingDay": "Jour de début",
+  "FSC.Configuration.Season.StartingDayHelp": "Le jour du mois où cette saison commence chaque année.",
+  "FSC.Configuration.Season.Color": "Couleur",
+  "FSC.Configuration.Season.ColorHelp": "La couleur à associer à cette saison.",
+  "FSC.Configuration.Season.ColorWhite": "Blanc",
+  "FSC.Configuration.Season.ColorSpring": "Printemps",
+  "FSC.Configuration.Season.ColorSummer": "Été",
+  "FSC.Configuration.Season.ColorFall": "Automne",
+  "FSC.Configuration.Season.ColorWinter": "Hiver",
+  "FSC.Configuration.Season.ColorCustom": "Couleur personnalisée",
+  "FSC.Configuration.Season.ColorCustomHelp": "Définir une couleur de valeur hexadécimale personnalisée pour la saison.",
+  "FSC.Configuration.Season.RemoveAll": "Supprimer toutes les saisons",
+  "FSC.Configuration.Season.Add": "Ajouter une nouvelle saison",
+  "FSC.Configuration.Month.Title": "Paramètres du mois",
+  "FSC.Configuration.Month.Name": "Nom du mois",
+  "FSC.Configuration.Month.NameHelp": "Le nom textuel de ce mois.",
+  "FSC.Configuration.Month.Days": "Nombre de jours",
+  "FSC.Configuration.Month.DaysHelp": "Le nombre de jours qui composent ce mois.",
+  "FSC.Configuration.Month.Intercalaire": "Mois Intercalaire",
+  "FSC.Configuration.Month.IntercalaryHelp": "Si ce mois doit être considéré comme un mois intercalaire ou non.",
+  "FSC.Configuration.Month.IntercalaryInclude": "Inclure le mois intercalaire dans les calculs de jour",
+  "FSC.Configuration.Month.IntercalaryIncludeHelp": "Si coché, le mois intercalaire sera traité comme un mois normal pour calculer le jour de la semaine et le nombre total de jours dans une année, mais sera toujours répertorié comme un mois intercalaire.",
+  "FSC.Configuration.Month.DayOffset": "Décalage du jour",
+  "FSC.Configuration.Month.DayOffsetHelp": "Le montant pour décaler le numéro du jour. Utilisé si vous voulez commencer un mois à 2 au lieu de 1",
+  "FSC.Configuration.Month.Number": "Numéro de mois",
+  "FSC.Configuration.Month.NumberHelp": "La représentation numérique de ce mois.",
+  "FSC.Configuration.Month.StartingWeekDay": "Démarre le jour de la semaine",
+  "FSC.Configuration.Month.StartingWeekDayHelp": "Choisissez par quel jour de la semaine ce mois commencera toujours, ou laissez par défaut pour laisser le calendrier déterminer le jour de la semaine approprié.",
+  "FSC.Configuration.Month.Remove": "Supprimer",
+  "FSC.Configuration.Month.RemoveAll": "Supprimer tous les mois",
+  "FSC.Configuration.Month.Add": "Ajouter un nouveau mois",
+  "FSC.Configuration.Months": "Mois",
+  "FSC.Configuration.Weekday.Title": "Paramètres des jours de la semaine",
+  "FSC.Configuration.Weekday.ShowHeadings": "Afficher les titres des jours de la semaine",
+  "FSC.Configuration.Weekday.ShowHeadingsHelp": "Si pour afficher les en-têtes des jours de la semaine sur le calendrier.",
```

Comment:
Should be "Cochez pour afficher......". I think the original string should be reworked too.